helm/slurm-cluster/values.yaml

```diff
@@ -104,6 +104,7 @@ slurmNodes:
   accounting:
     enabled: true
     k8sNodeFilterName: "no-gpu"
+    slurmdbdConfig: {}
```

Comment:
I think it's better to give a commented out example section of slurmdbd config here. Now it looks so ambiguous